docs/gallery/general/plot_configurator.py

```diff
@@ -0,0 +1,108 @@
+"""
+TypeConfigurator
+=======
+
+This is a user guide for how to take advantage of the
+:py:class:`~hdmf.term_set.TypeConfigurator` class by creating configuration files in
+order to more easily validate terms within datasets or attributes.
```

Comment:
```suggestion
Configuring PyNWB to allow only certain terms
===================================

This is a user guide for how to configure PyNWB to allow only certain terms for
particular fields within datasets and attributes using a configuration file and
the :py:class:`~hdmf.term_set.TypeConfigurator` class. This
approach can be easier than wrapping individual data objects with a
:py:class:`~hdmf.term_set.TermSetWrapper`.
```

Reply:
"Configuring PyNWB to allow only certain terms" - I don't like this title either. I think this is too broad. @mavaylon1 any ideas for a succinct but descriptive title?

Reply:
How about "How to Configure Term Validation"

Reply:
Sounds good